### ISSUE TITLE
Show serial number

### ISSUE THREAD

**Issue by giovannicaligaris** [elementary/switchboard-plug-about#43]
Would it be possible to show the machine's serial number in the About section?

sudo dmidecode -t system | grep Serial

**Comment by KeironO** [comment]
I've had a look at doing this, but I'm not sure we can as all of the methods of checking this sort of information requires sudo'ers access.

**Comment by arshubham** [comment]
I don't think this will be applicable to all devices, only prebuilt ones (probably only laptops). Mine shows `Serial Number: Default string` on a PC
